### ISSUE TITLE
Error while updating property 'disableIntervalMomentum' of a view managed by: RCTScrollView ANDROID

### ISSUE THREAD

**Issue by nguyenduy2392** [gorhom/react-native-bottom-sheet#12]
# Bug

## Environment info

| Library                         | Version |
| ------------------------------- | ------- |
| @gorhom/bottom-sheet         | ^1.1.0   |
| react-native                    | 0.63.1   |
| react-native-reanimated         | ^1.12.0   |
| react-native-gesture-handler    |^1.6.1   |

## Reproducible sample code

```
                <BottomSheet2
                    // ref={bottomSheetRef}
                    initialSnapIndex={0}
                    snapPoints={[300, contentHeight]}
                // onChange={handleSheetChanges}
                >
                    <BottomSheetFlatList
                        data={this.state.dataProvider.length ? this.state.dataProvider : NOT_FOUND}
                        keyExtractor={item => item.id}
                        initialNumToRender={10}
                        windowSize={20}
                        maxToRenderPerBatch={5}
                        renderItem={this._rowRenderer}
                        // {...(header && {
                        //   stickyHeaderIndices: [0],
                        //   ListHeaderComponent: header,
                        // })}
                        contentContainerStyle={styles.listContainer}
                    // focusHook={useFocusEffect}
                    />
                </BottomSheet2>
```

![1597291007908](https://user-images.githubusercontent.com/8022364/90092712-d7287b80-dd53-11ea-8696-78c1d5ad3f72.JPEG)

## My solution
in FlatList.tsx
change ```disableIntervalMomentum={disableIntervalMomentum}``` to ```disableIntervalMomentum={false}``` and it working

i try fix ```disableIntervalMomentum: Animated.Node<number>;``` to ```disableIntervalMomentum: Animated.Node<boolean>;```but it not working

**Comment by gorhom** [comment]
Thanks @nguyenduy2392 for reporting this bug! I’ll look into it shortly :)

**Comment by gorhom** [comment]
this should be resolved with [v1.1.1](https://github.com/gorhom/react-native-bottom-sheet/blob/master/CHANGELOG.md#v111) 👍
